decision_making.md

```diff
@@ -0,0 +1,45 @@
+
+All members of a decision making body are required to participate in at least 2/3 of formal votes of that decision making body per calendar year (teams can decide how to account for the specifics of this in low-activity projects, etc.). Members that have not met the 2/3 vote participation threshold for a year will automatically be asked to step down at the end of that year. Those individuals remain eligible to rejoin the decision making body in the future as they become available to participate at the required level. The quorum for all formal votes will be 50% and a "blank" option will always be included, with "blank" option counting towards the quorum but not included in totals for calculating results.
+
+
+**Recording.** Once a decision has been made during the consensus seeking phase, or by a vote, the decision making body will record the decision in an appropriate location that is as open as possible given the decision and work of the particular body.
+
+## Optional aspects of decision making
+
+While the previous section is prescriptive for all Jupyter teams, the following are recommended guidelines rather than mandatory procedures.
+
+- Do not call a vote to short-circuit an ongoing discussion that is still productive in terms of exploring ideas and feedback. Votes should be called only when discussion has explored the space and stakeholders have provided input.
+- During the consensus seeking phase, don’t interpret a lack of participation, discussion, or feedback as consensus or the lack thereof. Dissent can be silent and sometimes people are supportive but busy.
+- If you are interested in a decision being made, it is your responsibility to encourage voting member/stakeholders/community participation in the decision making process. If you can’t get such participation, you may want to hold off on doing any significant work on the matter.
+- Special guidelines for software projects making decisions:
+  - Add a link to this document in your issue and PR templates along with a link to the decision making body membership (see recommended language below).
+  - Create a “Consensus Seeking” and “Vote” tags to make it easy to find issues/PRs where a decision is being made.
+  - When a vote is called on GitHub, the sponsor should summarize the decision and paste a checklist of the voting members to use in voting in the top entry (description) of the issue/PR.
+  - When communications (in whatever number of channels) point to a discussion that requires a project-wide decision, and before major implementation work starts, a standalone issue should be opened that summarizes the issue, points to relevant prior discussions, and calls for an explicit vote/decision. This issue should be tightly scoped to the relevant decision only, and include the "Vote" tag.
```

Comment:
It would make sense to have a Vote specific group on Discourse as not all folks monitor all the orgs on GitHub that could possibly contain a Vote.

Reply:
I think this is a good idea to centralize crucial communications in one space. Another option could be to use the `enhancement-proposals` repository.

Whether it's Discourse, a specific repo, a mailing list, etc, there needs to be just a single place that SSC members are **required** to monitor that serves as the "source of truth" for decisions